.github/workflows/checks.yml

```diff
@@ -58,6 +58,11 @@ jobs:
       - name: Check API declarations and docs work as intended
         run: yarn api-check
 
+      - name: Check dotcom localizations
+        run: |
+          yarn build-i18n 
+          git diff --exit-code --quiet || (echo "Please run yarn build-i18n and commit the changes" && exit 1)
```

Comment:
can we scope the diff to the locales-compiled folder? just to avoid potential false positives if we introduce any tooling that modifies the working tree later